### DISCUSSION TITLE
请问bitmap_hash 把字符串存到bitmap后还能取出字符串吗

### DISCUSSION THREAD

**Discussion by qiaohanqing** [apache/doris#21028] [A - General / Q&A] [answered]
open_id存到bitmap里面了，怎么能完整的提取出来请问？
比如：select bitmap_to_string(bitmap_hash('nihao'));
我想取出来’nihao‘，但实际上取出来
3726836999

**Comment by xavierzhao** [comment]
`select bitmap_to_string(to_bitmap(1));
`

**Comment by yagagagaga** [comment, marked as answer]
You need to maintain a mapping table that requires at least two fields: `open_ id` and `bitmap_hash(open_id)`. Then you can join this mapping table to get open_id by bitmap_id.
